engine/access/rpc/rate_limit_interceptor.go

```diff
@@ -53,8 +53,8 @@ func NewRateLimiterInterceptor(log zerolog.Logger, apiRateLimits map[string]int,
 	}
 }
 
-// unaryServerInterceptor rate limits the given request based on the limits defined when creating the rateLimiterInterceptor
-func (interceptor *rateLimiterInterceptor) unaryServerInterceptor(ctx context.Context,
+// UnaryServerInterceptor rate limits the given request based on the limits defined when creating the rateLimiterInterceptor
```

Comment:
If this is going to be used for all the gRPC interfaces, we should move this somewhere common. Maybe `engine/rpc/common`?

Reply:
https://github.com/onflow/flow-go/pull/2505/commits/6fc22e9e58ffd509b2d49d44b6b084f800e99b93